sprokit/src/sprokit/pipeline_util/lex_processor.cxx

```diff
@@ -637,7 +639,7 @@ lex_processor::priv::
 current_loc() const
 {
   // Get current location from the include file stack top element
-  return kwiver::vital::source_location( m_include_stack.back()->m_filename,
+  return kwiver::vital::source_location( std::make_shared< std::string >( *(m_include_stack.back()->m_filename) ),
```

Comment:
This should have nothing to do with the python interface. The string is used locally to report error locations. in fact, the m_filename and related changes could be reverted to std::shared_ptr.